src/integrations/slack.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const axios = require('axios');
+const config = require('../../.config');
+
+module.exports = {
+    async sendMessage(jobs) {
```

Comment:
Minor nitpick: parameter name could be better. This parameter doesn't necessarily need to contain jobs, it can contain anything else. So better to call this something more generic, like `slackMessage` or something like that.